include/mav/Message.h

```diff
@@ -153,6 +154,35 @@ namespace mav {
             throw std::runtime_error("Unknown base type"); // should never happen
         }
 
+        uint64_t _computeSignatureHash48(const std::array<uint8_t, 32>& key) const {
+            // signature = sha256_48(secret_key + header + payload + CRC + link-ID + timestamp)
+            constexpr size_t maxSize = 32 + MessageDefinition::HEADER_SIZE +
+                                       MessageDefinition::MAX_PAYLOAD_SIZE +
+                                       MessageDefinition::CHECKSUM_SIZE + 1 + 6;
```

Comment:
What are the 32 and 1 + 6 here? Could you make them into constants, rather than magic values?

Reply:
There is a comment on the line above explaining what they are. But yes, I can make them into constants.

Reply:
Yes please

Reply:
Done!